app-frontend/src/app/services/vendor/planetLabs.service.js

```diff
@@ -20,12 +20,14 @@ export default (app) => {
         }
 
         filterScenes(apiKey, requestBody) {
+            // eslint-disable-next-line no-undef
```

Comment:
add `btoa` to the top of the file instead of this:
`/* globals btoa */
also - why do we need to add ':' to the end of the key?